src/OpenTelemetry/Trace/EnrichmentScope.cs

```diff
@@ -0,0 +1,100 @@
+﻿// <copyright file="EnrichmentScope.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+using System;
```

Comment:
nit: add a blank line above this?

Reply:
Dang I even tried to be mindful of this 😄 Updated.

Reply:
I'll add another cop before we GA (and yet another cop to check CR/LF/CRLF and literal `TAB`s as the current repo is quite messy).